README.md

```diff
@@ -1,4 +1,113 @@
+
+📈**Performance boost**: Finetuner significantly increases the performance of pretrained models on domain specific neural search applications.
+
+🔱 **Simple yet powerful**: Interacting with Finetuner is simple and seamless, and also supports rich features such as
+siamese/triplet loss, metric learning, self-supervised pretraining, layer pruning, weights freezing, dimensionality reduction, and much more.
+
+☁ **Finetune in the cloud**: Finetuner runs your fine-tuning jobs on [Jina Cloud](https://github.com/jina-ai/jcloud). You never have to worry about provisioning (cloud) resources! Finetuner handles all related complexity and infrastructure.
+
+<!-- end elevator-pitch -->
+
+## What is the purpose of Finetuner?
+
+Finetuner enables performance gains on domain specific neural search tasks by allowing you to fine-tune models in the cloud. We have conducted experiments on three neural search tasks in different domains to illustrate these performance improvements.
+
+| Task              | ndcg score (pre-trained)  | ndcg score (fine-tuned)   | Performance gains (∂) | Time cost |
+|-------------------|--------------------------:|--------------------------:|----------------------:|----------:|
+| text-to-text      |                           |                           |                       |           |
+| image-to-image    |                           |                           |                       |           |
+| text-to-image     |                           |                           |                       |           |
+
+Finetuner also aims to make fine-tuning simple and fast. When interacting with Finetuner, the API takes care of all your fine-tuning jobs in the cloud. This only requires a few lines of code from you, as demonstrated in section 
+
+## How does it work?
+
+<img src="https://github.com/jina-ai/finetuner/blob/docs-update-readme/docs/_static/finetuner-client-journey.svg?raw=true" title="Finetuner Client user journey.">
+
+
+## Install
+
+Requires Python 3.7+ installed on Linux/MacOS.
+
+```bash
+pip install -U finetuner-client
+```
+
+
+## Fine-tuning ResNet50 on Totally Looks Like dataset
+
+```python
+import finetuner
+from finetuner.callback import BestModelCheckpoint, EvaluationCallback
+
+finetuner.login()
+
+finetuner.create_experiment(name='tll-experiment')
+
+run = finetuner.fit(
+        model='resnet50',
+        experiment_name='tll-experiment',
+        run_name='resnet-tll',
+        description='fine-tune the whole model.',
+        train_data='resnet-tll-train-data',
+        eval_data='resnet-tll-eval-data',
+        loss='TripletMarginLoss',
+        callbacks=[BestModelCheckpoint(), EvaluationCallback(query_data='resnet-tll-eval-data')],
+        epochs=6,
+        learning_rate=0.001,
+    )
+
+print(run.status())
+print(run.logs())
+
+run.save_model('resnet-tll')
```

Comment:
can we make it minimal, do not include the arguments which can be fileed automatically, such as experiment name and run name